### ISSUE TITLE
Fit & finish - UI Copy | Migration to Composer 2.0

### ISSUE THREAD

**Issue by mareekuh** [microsoft/BotFramework-Composer#7600]
Fit & finish - UI Copy | Migration to Composer 2.0

White cells = existing copy
Yellow/blue cells = new copy

Link to 'live' copy on ' Migration ' tab: 
https://microsoft.sharepoint.com/:x:/r/teams/ConversationalAI785/_layouts/15/Doc.aspx?sourcedoc=%7B6B1FACBD-B5FE-4AD2-A35B-995635C1696B%7D&file=glossary.xlsx&action=edit&mobileredirect=true&wdPreviousSession=36539654-a42b-4b87-be1b-de96507b4422&wdOrigin=TEAMS-ELECTRON.teams.undefined&cid=676cf454-61e0-4182-bd27-39044bb51cea

![Migration_1](https://user-images.githubusercontent.com/48503434/117046537-a6afb580-acc5-11eb-8ef8-9f169307dd2c.png)
![Migration_2](https://user-images.githubusercontent.com/48503434/117046547-a8797900-acc5-11eb-9dd2-be1e0beb9c6b.png)

**Comment by mewa1024** [comment]
@mareekuh I added a couple comments/questions in the Excel file:
- "update" or "upgrade"? I was using "update" since I think "update" implies a new version you get for free whereas "upgrade" is more better stuff you pay for.
- for line 47, the modal before you convert your project, I think we should be explicit that we are creating a copy of the original so people are reassured that the original project will not be written over
- for line 60, can we use the message text in the green column? It's meant to guide users to what to do next.

**Comment by sgellock** [comment]
Shiproom - approved WITH CONDITIONS

this came in after we agreed to the scope of today's fit and finish work.  IF we can get this done AND merged by EOD today, then we'll take it.  If not, it will get moved out to R14

**Comment by beyackle** [comment]
I don't think this is going to make it in today. The other changes have had enough ambiguity and technical problems getting merged.
